backend/src/create.rs

```diff
@@ -481,3 +496,23 @@ fn nodes_to_edges(graph: &Graph, nodes: Vec<utils::osm2graph::IntersectionID>) -
     }
     edges
 }
+
+fn get_poi(tags: &Tags, point: Coord) -> Option<POI> {
+    if tags.is_any("shop", vec!["convenience", "grocery", "supermarket"]) {
+        return Some(POI {
+            point: point.into(),
+            kind: POIKind::Grocery,
+            name: tags.get("name").cloned(),
+        });
+    }
+
+    if tags.is_any("amenity", vec!["community_centre", "library"]) {
```

Comment:
I'm taking the liberty of lumping libraries into this category too, but may change after discussing with Cici